style.css

```diff
@@ -0,0 +1,415 @@
+body {
+  min-width: 375px;
+  margin: 0;
+  padding: 0;
+  font-family: Montserrat, Arial, sans-serif;
+}
+
+.header {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+    -webkit-box-pack: justify;
+    -ms-flex-pack: justify;
+  justify-content: space-between;
+  min-height: 70px;
+    -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+.header__lang {
+  display: none;
+}
+
+.header__logo {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  text-decoration: none;
+  position: absolute;
+  width: 98px;
+  height: 37px;
+  margin-left: 29px;
+  margin-top: 22px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 30px;
+  line-height: 37px;
+  text-align: center;
+  letter-spacing: 0.07em;
+  color: #fff;
+}
+
+.header__menu {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -ms-flex-item-align: end;
+  align-self: flex-end;
+  width: 70px;
+  height: 70px;
+  background: #f5f5f3;
+  cursor: pointer;
+}
+
+.header__burger {
+  -ms-flex-item-align: center;
+  -ms-grid-row-align: center;
+  align-self: center;
+  margin: auto;
+}
+
+.header__burger-line {
+  display: block;
+  width: 26px;
+  height: 3px;
+  background: #000000;
+}
+
+.header__burger-line:first-of-type {
+  margin-bottom: 5px;
+}
+
+button {
+  background: none;
+}
+
+.up-section__btn_else,
+.up-section__btn_offer {
+  text-transform: uppercase;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+}
+
+.up-section__btn.up-section__btn_else {
+  width: 203px;
+  height: 30px;
+  padding-top: 14px;
+  border: 1.5px solid #000000;
+  justify-content: center;
+}
+
+.up-section__btn_else:hover {
+  background-color: #eeddd0;
+}
+
+.up-section {
+  position: relative;
+  width: 100%;
+}
+
+.up-section__title {
+  position: relative;
+  z-index: 10;
+  padding-left: 26px;
+  margin-top: -165px;
+  font-weight: bold;
+  line-height: 51px;
+  font-size: 50px;
+}
+
+.up-section__description,
+.up-section__contacts {
+  display: none;
+}
+
+.up-section__images {
+  z-index: -10;
+  height: 529px;
+  overflow: hidden;
+}
+
+.up-section__collection-image img {
+  width: 100%;
+  height: 100%;
+      -o-object-fit: cover;
+  object-fit: cover;
+      -o-object-position: 0 -55px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/locoalin/pullRequest?prid=5676737)